exasol_script_languages_developer_sandbox/lib/deployer.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2012-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
hm, we have this now already in two projects. I am thinking, about if we should move this to its own package

Reply:
Hm, maybe something like boto3-extionsion, or aws-python-access